src/idsk/components/customer-surveys/template.njk

```diff
@@ -0,0 +1,214 @@
+<div data-module="idsk-customer-surveys" class="idsk-customer-surveys">
+  <div class="govuk-width-container">
+    <div class="idsk-customer-surveys__logo">
+      {% include('../../../idsk/assets/images/logo-mirri-farebne-sk-2.svg') %}
+    </div>
+    <div id="idsk-customer-surveys__content">
+      {#1st#}
+      <div class="idsk-customer-surveys__step idsk-customer-surveys--show">
+        <p class="govuk-body-l">
+          Cieľom tohto dotazníka je zistiť potreby a postrehy používateľov webu Ministerstva investícií, regionálneho rozvoja a informatizácie (Ďalej len "MIRRI).
```

Comment:
otazne je tiez toto.. ale pravdepodobne aj otazky by mali byt paramerizovane

Reply:
vsetky otazky su parametrizovane. len tieto uvodne 3 vety niesu, daju sa paramerizovat, otazne je ze ako. Na 3 <p> tagy ? alebo ako jeden kde bude povedane ako maju byt oddelene ?

Reply:
mozme to spravit nejak tak ze:
descriptions: [
   description1: 'xx',
   description2: 'yy',
   description3: 'zz'
]